app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsController < ApplicationController
+    def index
+        @keyword = params[:'q']
+        @foods = FoodFacade.food_search(@keyword).first(10)
```

Comment:
If you include params for `:totalHits` and set it to 10 in your service, you won't need to use `.first(10)`
Think about how you could refactor this to make one call to the Facade and get both count and food results.